docs/gambit/gambit.md

```diff
@@ -0,0 +1,106 @@
+# Gambit: Certora's Mutation Generator for Solidity
+
+This is a mutation generator for Solidity.
+Mutation Testing is a technique for
+  evaluating and improving test suites or specifications used
+  for testing or verifying Solidity smart contracts.
+
+Gambit traverses the Solidity AST generated by the Solidity compiler
```

Comment:
```suggestion
Gambit traverses the Solidity [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) generated by the Solidity compiler
```
I think readers of this page are likely to have diverse levels of familiarity with various CS concepts. I would consider linking some explanation of what an AST is. I chose wikipedia, but there might be something better.